httpserver/onion_processing.cpp

```diff
@@ -317,8 +246,8 @@ void RequestHandler::process_onion_req(const std::string& ciphertext,
         // Forward the request to url but only if it ends in `/lsrpc`
         if ((target.rfind("/lsrpc") == target.size() - 6) &&
```

Comment:
Is it feasible to tighten up this target requirement?  I imagine there could be some site out there hosting relative large files that uses non-query-string parameters (but still sends a large file) that someone could abuse by spamming an endless number of http proxy requests via the network's 1500 storage servers.  If we tighten it to a list of fixed acceptable endpoint names that concern goes down greatly.

Reply:
Are you suggesting something like `target == "/loki/v3/lsrpc"`? I don't know if we want to allow the changes in the version number at least. How about `starts_with("/loki/") && ends_with("lsrpc")`? Do you have any input on this @nielsandriesse?

Reply:
> How about starts_with("/loki/") && ends_with("/lsrpc")

That would be better, for sure.

Reply:
Updated with `/loki/` and `/oxen/` as prefixes.